### PR TITLE
Add support for Scala 2.13.0

### PR DESCRIPTION
**Pull request by lustefaniak**

### CODE REVIEW

**Comment by lustefaniak** [comment]
Requires https://github.com/kamon-io/kamon-futures/pull/11 release

**Comment by ivantopo** [comment]
@lustefaniak I'll release that a bit later today and ping you when done :)

**Comment by ivantopo** [comment]
Hey @lustefaniak, just released the futures module as 2.0.0-RC2!

**Comment by lustefaniak** [comment]
@ivantopo Looks green ;)

File: build.sbt [hunk 1]
```diff
@@ -16,10 +16,10 @@
...
 val kamonCore       = "io.kamon" %% "kamon-core"         % "2.0.0-RC1"
 val kamonTestkit    = "io.kamon" %% "kamon-testkit"      % "2.0.0-RC1"
-val kamonScala      = "io.kamon" %% "kamon-scala-future" % "2.0.0-RC1"
-val kamonExecutors  = "io.kamon" %% "kamon-executors"    % "2.0.0-RC1"
-val kamonInstrument = "io.kamon" %% "kamon-instrumentation-common" % "2.0.0-RC1"
-val kanelaAgent     =  "io.kamon" % "kanela-agent"       % "1.0.0-M3"
+val kamonScala      = "io.kamon" %% "kamon-scala-future" % "2.0.0-RC2"
+val kamonExecutors  = "io.kamon" %% "kamon-executors"    % "2.0.0-RC2"
+val kamonInstrument = "io.kamon" %% "kamon-instrumentation-common" % "2.0.0-RC2"
+val kanelaAgent     =  "io.kamon" % "kanela-agent"       % "1.0.0-RC3"
```
Comment (dpsoft) [inline build.sbt]: `kanela-agent` dependency should be `1.0.0-RC4`

Comment (lustefaniak) [reply build.sbt]: done

**Pull request merged**